### DISCUSSION TITLE
The language server crashed 5 times in the last 3 minutes. It will not be restarted.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-cpptools#8196] [Q&A]
Hi, I am getting this error after updating C/C++ extensions that were recommended in VS Code. 
This is the message "The language server crashed 5 times in the last 3 minutes. It will not be restarted."

Any help would be great!

**Comment by sean-mcmanus** [comment]
What OS and C/C++ extension version are you on? If you're on 1.7.0-insiders with Linux x64 Ubuntu 14/16, Debian 9, or CentOS 7 then you're likely hitting https://github.com/microsoft/vscode-cpptools/issues/8192, which we have a pending fix for 1.7.0-insiders2. The workaround is to change C_Cpp.updateChannel to "Default" until we release 1.7.0-insiders2.

We also have another report for Debian 10 at https://github.com/microsoft/vscode-cpptools/issues/8201, but we haven't determined what the cause of that is yet (same workaround works though).

**Reply by MNoumanAbbasi** [reply]
Running on 1.60.2 and using Remote SSH. The extension works locally but gives the same "The language server crashed ..." error when enabling the extension on remote SSH.

**Reply by sean-mcmanus** [reply]
@MNoumanAbbasi 1.60.2 is the VS Code version. What is your target/remote OS and C/C++ extension version? If it repros with 1.7.0-insiders2, you should probably file a bug report: https://github.com/microsoft/vscode-cpptools/issues/new?assignees=&labels=&template=language-service.md&title=
